uwblueprint/website-bp-be
# Seed application data from JSON file

## Notion ticket link

[[BE] Create test data](https://www.notion.so/uwblueprintexecs/BE-Create-test-data-8b409d6acb8945bf923a99919ab4f7a9)



## Implementation description
* Created a function to get data from a JSON file when calling `node migrate up` for application info. 
* Removed hardcoded SEEDED_DATA in migration code. 



## Steps to test
1. Run `node migrate down` enough times to get rid of the `applicantresponse` table. You may need to remove the whole Docker container if the migration doesn't cooperate. 

2. From `docker-compose exec ts-backend bash`, run `node migrate up`. 

3. From the Postgre table (`docker-compose exec db bash && psql -U postgres -d recruitment_tools`), check the applicantresponse table (i do not recommend doing `SELECT * ...`) to see that the table is populated. 



## What should reviewers focus on?
* Implementation of the importing function is a little hardcoded right now, we might want to do that a little cleaner, but there is a bit of variation in how the fields need to be converted. 


## Checklist
